src/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/CustomValidationAttribute.cs

```diff
@@ -83,8 +83,7 @@ public sealed class CustomValidationAttribute : ValidationAttribute
         ///     <see cref="Method" />.
         /// </param>
         /// <param name="method">The name of the method to invoke in <paramref name="validatorType" />.</param>
-        public CustomValidationAttribute(Type validatorType, string method)
-            : base(() => SR.CustomValidationAttribute_ValidationError)
+        public CustomValidationAttribute(Type validatorType, string method) : base(SR.CustomValidationAttribute_ValidationError)
```

Comment:
Please revert this.  Constructing a CustomValidationAttribute now invokes SR.CustomValidationAttribute_ValidationError, whereas previously it didn't.

Reply:
I'm not really sure why this is bad - i was just being consistent with other attributes in the library - e.g. `CompareAttribute` and `MinLengthAttribute` and `MaxLengthAttribute`. However, I did revert these changes

Reply:
> I'm not really sure why this is bad

Because SR.ResourceString is not cheap; this change makes the constructor 5x more expensive.  Now maybe that's a good thing because it in turn reduces the cost of calling the relevant property's get accessor, but we don't know that: without data that shows moving these costs around is a good thing, we shouldn't be doing it.